perception/conda/recipe/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "librealsense" %}
```

Comment:
We're going to have more than librealsense for the perception conda environment, right?